### DISCUSSION TITLE
SFe 4.0 Release Candidate 1

### DISCUSSION THREAD

**Discussion by sylvia-leaf** [SFe-Team-was-taken/SFE#63] [Announcements]
# SFe 4.0 Release Candidate 1 (4.0-rc1)

## The first release candidate!

This is our first ever Release Candidate specification for SFe! We're so excited.

In no time, we will be ready to release the first version of the SFe specification. We hope that everyone enjoys it!

### Reformatting in progress

Compared to the previous draft milestone that we released yesterday, the structure has been completely changed based on feedback by Spessasus. This structure makes it easier for people to read, because it skips as much of the unnecessary information as possible. While this may sound counter-intuitive, we will provide the detailed description of everything that is added to SFe in future versions. For example, the requested detailed modulator implementation will be added for 4.1, as that is planned to be the modulator update, and things will be completely different from legacy SF2.04.

The only thing that we haven't done yet is the "nice-looking" version of the specification, but that will come in time. It will also come before RC2 is available.

### Adding stuff about release candidates

We also added the option of "release candidate" for `SFvx`, because this was required for this new versioning concept.

Yeah, I know that we said no more versioning updates, but this was necessary because we weren't quite ready to release the final specification!

### About future plans

The future plans were already in the `4.0.11` release notes.

### Why are the release notes so short?

In short, because we released the previous draft specification, `4.0.11` yesterday!

We just managed to get this re-arranged specification ready in no time.

---

## What's Changed
* Release Candidate 1 by @sylvia-leaf in https://github.com/SFe-Team-was-taken/SFe/pull/62

**Full Changelog**: https://github.com/SFe-Team-was-taken/SFe/compare/4.0.11...4.0-rc1

<hr /><em>This discussion was created from the release <a href='https://github.com/SFe-Team-was-taken/SFe/releases/tag/4.0-rc1'>SFe 4.0 Release Candidate 1</a>.</em>

**Comment by sylvia-leaf** [comment]
Release candidate 2 is planned for 11 January 2025.
